lib/svg-sprite.js

```diff
@@ -30,7 +29,8 @@ const CPU_COUNT = os.cpus() && os.cpus().length;
 /**
  * SVGSpriter class
  *
- * @param {Object} config                Configuration
+ * @param {object} config                Configuration
+ * @class
```

Comment:
Not sure about this

Reply:
It's not a Class AFAICT. I tried in the past to switch to Classes but didn't succeed so I kept the prototype methods.